app/metrics.yaml

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+toolbar.events:
+  url_committed:
+    type: event
+    description: >
+      This metric is recorded when a URL is committed (loaded) from the browser toolbar component.
```

Comment:
No need to include the link! The main thing is to include all possible values that could be sent, and any possible extras.

Something like:
"Records `toolbar` when a url is committed (loaded) from the browser toolbar component"
and could optionally clarify what cases (if that's known, like via keyboard or UI, or if it could be anything)

Sorry for the confusion!

Reply:
No, thank you!  This is my first go around instrumenting a metric (and my first data-review) so I appreciate the feedback 😃